### ISSUE TITLE
snap effects multiples layers

### ISSUE THREAD

**Issue by atavoidturk** [mbloch/mapshaper#636]
Snap effects layers with the same source, even if the target is explicitly specified with target=

**Comment by mbloch** [comment]
You are right, but it's not a simple thing to fix. Sometimes, layers can share the same set of coordinate data, for example, if they are derived from the same source layer, or if a group of files are imported together, using the `combine-files` flag. Certain commands will modify the coordinates of all layers in the group, even if you only specify one of the layers as a target. `-simplify` works this way, and so does `-snap`. Most of the time this is the behavior that I want... but this behavior can also take users by surprise. I haven't figured out how to explain the layer grouping concept to users in a clear and simple way. I welcome any suggestions that you have.
